### DISCUSSION TITLE
BUG - Confirmation

### DISCUSSION THREAD

**Discussion by Voyrox** [anuraghazra/github-readme-stats#2802] [Q&A] [answered]
This is the code for my GitHub section `https://github-readme-stats.vercel.app/api?username=Frost-Lord&count_private=true&show_icons=true&hide_title=true&include_all_commits=true&theme=dark&bg_color=0c1014`

![image](https://github.com/anuraghazra/github-readme-stats/assets/51778028/b7a0f1e3-ca60-4f67-a0d4-61d32ee12133)

But the problem is that it's saying I have 461 GitHub comments which are wrong as due to an update on your end or something I went from over 1k down to 461 comments or idk what happened. Could anyone check if what I have done is correct or did something change?

**Comment by qwerty541** [comment, marked as answer]
Recently we merged #2736 pull request which removes `count_private` query option due to its unexpected and confusing behavior. Currently to show private contributions on your stats card you should [deploy your own instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) using your own GitHub API token.

**Comment by NikoEscobar** [comment]
I'm experiencing a similar issue. I conducted a direct test using the GitHub API, and it returned the exact same value as what's being displayed on github-readme-stats. I'm using my own instance and I even granted full access to my personal token, but the results remained unchanged. It seems that this issue is not specific to open or private repositories, but rather to projects that have been moved from GitHub and subsequently deleted. However, it's confusing because the main contribution panel on our profiles is still accurately computing the commits. Any insights or suggestions would be greatly appreciated.

**Reply by rickstaa** [reply]
@NikoEscobar I checked the results for my account and everything is working fine on the Public and private instance. The card will display the exact number of commits given by [the GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer). I think you are confusing commits with contributions, which also includes other GitHub interactions (see https://github.com/anuraghazra/github-readme-stats/discussions/1679). As stated in https://github.com/anuraghazra/github-readme-stats/discussions/2802#discussioncomment-6141639, we recently fixed a bug which added contributions to the commits' stat. Although, when public on your profile, these contributions will include your private commits they will also include other contributions like PRS, issues, discussion posts etc. Because this led to incorrect stats, we decided to remove the `include_private` flag and instead let people deploy their own Vercel instance (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own). I would advice you to deploy your own instance if you want your private commits to show up on the card.

You check whether the card displays the right number of commits by using the following syntax in the [GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer):


```graphql
query userInfo($login: String!) {
  user(login: $login) {
    name
    login
    contributionsCollection {
      totalCommitContributions
      restrictedContributionsCount
    }
  }
}
```

```graphql
{"login": "rickstaa"}
```

Please bear in mind that since you are logged in, the GraphQL explorer will also show private commits where the public instance will not.

#### Niko's public results

**GraphQL output

```graphql
{
  "data": {
    "user": {
      "name": "Níko Escobar",
      "login": "NikoEscobar",
      "contributionsCollection": {
        "totalCommitContributions": 2,
        "restrictedContributionsCount": 1845
      }
    }
  }
}
```

**The public card**

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=NikoEscobar)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=NikoEscobar)](https://github.com/anuraghazra/github-readme-stats)
```

**Reply by NikoEscobar** [reply]
I see now, I was indeed mistaken contributions with commits, thank you =)

**Reply by rickstaa** [reply]
No problem the mistake was also present in the old 😅.
